docker-config/os-packager/Dockerfile

```diff
@@ -1,12 +1,17 @@
-FROM gliderlabs/alpine:latest
+FROM gliderlabs/alpine:3.3
+
 
 RUN apk add --update python
+RUN apk add --update --virtual=build-dependencies ca-certificates build-base
 RUN apk add git
 RUN apk add nodejs
 RUN git clone http://github.com/openspending/os-packager.git app
+RUN npm install npm -g
+RUN npm --version
 RUN cd app && npm install --only=prod
```

Comment:
Why are you separating the dev and production builds for npm? Just for debugging? Also, we should not ever need to install dev dependencies.